Basic-Car-Maintenance/Documentation.docc/getting-started.tutorial

```diff
@@ -127,6 +127,7 @@
                 `brew install openjdk` 
             }
             @Step {
+                The latest version of firebase-tools requires a JDK of 11 or greater. You can download the latest version here [JDK23](https://www.oracle.com/java/technologies/downloads/#jdk23-mac)
```

Comment:
What is the difference in using `brew install openjdk` vs what you get by going directly to download JDK23?

Reply:
According to the OpenJDK website, the install should install the JDK23 binaries. After some research it appears that in my case the 'brew install openjdk' failed to install the binaries so I had to do it manually for it to work. This ticket should be closed as the changes to the documentation are unnecessary. If contributors run into my issue we can refer them to the solution in this discussion: https://github.com/mikaelacaron/Basic-Car-Maintenance/discussions/317

Reply:
I think this is overall still a valid change, but worded differently it would make more sense
Because right now it looks like you have to install it twice, which *may not* be true for everyone, but adding an extra note of like "If you have issues related to Java not being installed you may have to install it directly without homebrew using this link ___"

but yeah removing all the `-` back to `*` does need to be changed back

Reply:
It seems that all the * were changed to -by VS Code (I assume) because I did not change them, and the original file has * instead of -. It also seems like VS Code added the extra spacing as well when I cut and pasted my content. So should I just cut and paste the contents of the original file from main into file in my branch to fix the * and spacing, and then make the other changes?

Reply:
Yes, you change all the `-` back to `*` and then commit and push (to the branch you're on, `rramirez-dev:324-improve-firebase-docs`) and the PR will update automatically

then click re-review button so I know it's ready to be reviewed again